src/editor.html

```diff
@@ -1696,6 +1710,10 @@
 				padding-left: 10px;
 				padding-right: 10px;
 			}
+
+			#zss_editor_footer {
+				height: 10px;
```

Comment:
Since this component is supposed to be used in various places we don't want to break the functionality. Consider a case for example that this editor is not full screen but is just showing a few lines, or any other use-case.

If you can, please specify this as a prop like you suggested and the default would remain the same as now (0).